ui/src/app/pages/analysis/runtime-panel.tsx

```diff
@@ -245,8 +246,13 @@ export const ConfirmDeleteUnattachedPD = ({onConfirm, onCancel}) => {
 
     <div style={styles.confirmWarning}>
       <h3 style={{...styles.baseHeader, ...styles.bold, gridColumn: 1, gridRow: 1}}>
-        <label ><input type= 'radio' checked={deleting === true} onChange={() => setDeleting(true)}/>
-          Delete persistent disk</label></h3>
+        <div key='Delete persistent disk' style={{display: 'inline-block', marginRight: '0.5rem'}}>
```

Comment:
key is mostly used for distinguishing elements of a list

To get an element via Puppeteer, I would either use the label text, or introduce a `data-test-id` here. @aweng98 may be able to advise on which approach is preferred.

Reply:
attribute `data-test-id` is preferred for e2e automation.